### PR TITLE
UI add amendment confirmation certify component.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#9606

*Description of changes:*
- Add certify information to amendment confirmation.
- 9608 UI load crown charge secured parties with new API endpoint.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by kialj876 — APPROVED** [review]
Looks good to me, nice work!

File: ppr-ui/src/components/common/CertifyInformation.vue [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      certified: false,
+      showErrors: props.setShowErrors,
+      showErrorComponent: computed((): boolean => {
+        return (localState.showErrors && !localState.valid)
+      }),
+      certifyInformation: null,
+      currentDate: computed((): string => {
+        return convertDate(new Date(), false, false)
+      }),
+      warningText: computed((): string => {
```
Comment (jordiwes) [inline ppr-ui/src/components/common/CertifyInformation.vue]: curious why this is computed...

File: ppr-ui/src/components/common/CertifyInformation.vue [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                localState.legalName &&
+                localState.legalName.trim().length > 1 &&
+                localState.legalName.length <= 100)
+      }),
+      rules: [
+        (v: string) => !v || v.length <= 100 || 'Maximum 100 characters reached' // maximum character count
+      ]
+    })
+
+    watch(() => props.setShowErrors, (val) => {
```
Comment (jordiwes) [inline ppr-ui/src/components/common/CertifyInformation.vue]: you can make localstate.showErrors just a computed property on this prop

File: ppr-ui/src/components/common/CertifyInformation.vue [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+      ...toRefs(localState)
+    }
+  }
+})
+</script>
+
+<style lang="scss" module>
+@import '@/assets/styles/theme.scss';
+
+.certify-checkbox {
```
Comment (jordiwes) [inline ppr-ui/src/components/common/CertifyInformation.vue]: is this class used?

File: ppr-ui/src/components/common/CertifyInformation.vue (hunk 1 above)
Comment (doug-lovett) [reply ppr-ui/src/components/common/CertifyInformation.vue]: You're right it's just static text. Change to a regular property.

File: ppr-ui/src/components/common/CertifyInformation.vue (hunk 2 above)
Comment (doug-lovett) [reply ppr-ui/src/components/common/CertifyInformation.vue]: Change made.

File: ppr-ui/src/components/common/CertifyInformation.vue (hunk 3 above)
Comment (doug-lovett) [reply ppr-ui/src/components/common/CertifyInformation.vue]: It was but no longer - I have removed it.

**Review by jordiwes — APPROVED** [review]
Great!

**Pull request merged**